domains/u.json

```diff
@@ -0,0 +1,11 @@
+{
+        "owner": {
+           "username": "ImShyMike",
+           "discord": "582648583635992622"
+        },
```

Comment:
missing mail field

Reply:
do I need to use the owl token?

Reply:
You just need to add the email field but leave it blank.